runtime/integration-tests/src/genesis.rs

```diff
@@ -0,0 +1,144 @@
+use crate::*;
+// Substrate
+use sp_core::{ storage::Storage};
+
+// Cumulus
+use integration_tests_common::{
+    constants::{accounts, collators},
+};
+
+pub const SAFE_XCM_VERSION: u32 = 3;
+
+use crate::mock::units;
+
+
+
+#[macro_export]
+macro_rules! genesis_gen {
+	($runtime:ident, $para_account_id: ident, $asset_metadata: ident) => {{
+		use $runtime::BuildStorage;
+        use crate::mock::units;
+        use integration_tests_common::constants::{collators, accounts};
+        use spacewalk_primitives::CurrencyId;
+        pub const SAFE_XCM_VERSION: u32 = 3;
```

Comment:
Do we need to define this twice? It's already defined in l. 10.

Reply:
Nope! I'll keep the l. 23 definition.